src/test/java/seedu/address/model/appointment/AppointmentContainsPatientPredicateTest.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.model.appointment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalAppointments.APPOINTMENT_1;
+import static seedu.address.testutil.TypicalAppointments.APPOINTMENT_4;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class AppointmentContainsPatientPredicateTest {
+    @Test
+    public void equals() {
```

Comment:
Perhaps it might be more readable to split this into multiple smaller methods with different input types and expected outputs?